repos/system_upgrade/common/actors/checkfips/actor.py

```diff
@@ -33,3 +37,11 @@ def process(self):
                     reporting.Groups([reporting.Groups.SECURITY]),
                     reporting.Groups([reporting.Groups.INHIBITOR])
                 ])
+        else:
+            if fips_info.is_enabled:
+                fips_required_initramfs_files = [
+                    '/usr/lib64/ossl-modules/fips.so',
+                    '/etc/pki/tls/openssl.cnf',
```

Comment:
The default `/etc/pkit/tls/openssl.cnf` includes `/etc/crypto-policies/back-ends/opensslcnf.config` and will fail if that file isn't present, so we'll also need that.

On the other hand, this is clearly a bug in dracut (see https://bugzilla.redhat.com/show_bug.cgi?id=2176560), so maybe we should just wait for dracut to fix this, rather than working around it here?

Reply:
I agree that we should not be copying the files ourselves, and the code copying the files will be removed once the problem is fixed in dracut.  I've opted for implementing the workaround as it did not require too much (temporary) code and it would allow for further prototyping if need be.

Reply:
I've added the `opensslcnf.config` into the list of files we include in the initramfs.